ydb/src/grpc_wrapper/raw_topic_service/common/codecs.rs

```diff
@@ -0,0 +1,26 @@
+use itertools::Itertools;
+use ydb_grpc::ydb_proto::topic::SupportedCodecs;
+
+#[repr(i32)]
+#[derive(serde::Serialize)]
+pub(crate) enum RawCodec {
```

Comment:
1. Custom is only start point of customer codecs, not fixed value.
The codec value can be 10001 or 12345.

2. When we will read grpc stream it can contain values, not specified by current proto-enumeration and we need to handle it.

What if:
1. Do custom variant with value? Custom(i32) and store all unknown value to the custom field
2. remove repr directive - it will bad for nested values and any way we no need any special layout.

or store RawCodec as struct with only one i32 field + some constants for well known values?